app/controllers/concerns/emit_request_events.rb

```diff
@@ -15,7 +15,19 @@ def trigger_request_event
         .with_user_and_namespace(current_user, current_namespace)
         .with_request_uuid(RequestLocals.fetch(:request_id) { nil })
 
-      SendEventsToBigquery.perform_async(request_event.as_json)
+      EM.next_tick do
```

Comment:
This looks promising. Please could we feature flag it, and deduplicate it? Either into SendEventstoBigquery itself, or into a generic wrapper. (I think I'd favour the former, for now?)

Reply:
Also is it worth targeting `load-test` rather than `main` at first?